docs/connectors/organizations.md

```diff
@@ -0,0 +1,90 @@
+# Organizations Connector
+
+This is the main connector of the Aragon Connect library. It is responsabile to parse the organizations data.
+
+Currently a single flavor of this connector is avaiable, build-in the core library using The Graph. We have plans to keep including other flavors, like a Ethereum connector that reduce the state from blockchain events, or a SQL connector that fetch data from a database, etc.
+
+## Connector Interface
+
+The connector to be compatible with Aragon Connect library should implement the following interface:
+
+```typescript
+  chainId?: number
+  permissionsForOrg(orgAddress: string): Promise<Permission[]>
+  onPermissionsForOrg(orgAddress: string, callback: Function): { unsubscribe: Function }
+  appsForOrg(orgAddress: string): Promise<App[]>
+  onAppsForOrg(orgAddress: string, callback: Function): { unsubscribe: Function }
+  repoForApp(appAddress: string): Promise<Repo>
+  appByAddress(appAddress: string): Promise<App>
+  rolesForAddress(appAddress: string): Promise<Role[]>
+```
+
+## The Graph Connector
+
+This connectors is build using The Graph and realaying on the GraphQL technology to query the data.
+
+### GraphQLWrapper
+
+The Graph connector export the `GraphQLWrapper` object. The wrapper is useful when you want to create low level request and talk to GraphQL directly.
```

Comment:
Suggesting to change `GraphQL` to `subgraph` as we're talking to that; graphql is just the query language used.
```suggestion
The Graph connector exports the `GraphQLWrapper` object. The wrapper is useful when you want to create low level requests and talk to the subgraph directly.
```